zerver/context_processors.py

```diff
@@ -152,13 +152,29 @@ def login_context(request: HttpRequest) -> Dict[str, Any]:
         realm_description = get_realm_rendered_description(realm)
         realm_invite_required = realm.invite_required
 
+    def get_apple_locale() -> str:
+        supported_locales = ['ar_SA', 'ca_ES', 'cs_CZ', 'da_DK', 'de_DE',
+                             'en_US', 'es_ES', 'es_MX', 'fi_FI', 'fr_FR',
+                             'hr_HR', 'hu_HU', 'el_GR', 'nl_NL', 'no_NO',
+                             'it_IT', 'iw_IL', 'ja_JP', 'ko_KR', 'ms_MY',
+                             'pt_PT', 'ro_RO', 'ru_RU', 'sk_SK', 'sv_SE',
+                             'pl_PL', 'th_TH', 'tr_TR', 'uk_UA', 'vi_VI'
+                             'zh_CN', 'zh_HK', 'id_ID', 'pt_BR']
```

Comment:
This should get a comment saying where it comes from -- in particular a link to https://developer.apple.com/documentation/signinwithapplejs/incorporating_sign_in_with_apple_into_other_platforms .

I'm also kind of confused by how it's mostly, but not quite, in order. How about putting it in the same order it appears in on that docs page? That'll make it easier to scan with the eye to see they match.

Reply:
> I'm also kind of confused by how it's mostly, but not quite, in order. 

That's because I shuffled just some at end of each line to have equal line length for all the lines.

I've put them in order and removed these values `'en_GB', 'fr_CA', 'es_MX', 'pt_BR', 'zh_HK',zh_TW'` as they have same language code, thinking these are less popular than those left in the list